### ISSUE TITLE
подключение модуля во время выполнения

### ISSUE THREAD

**Issue by kompolom** [bem/bem-forum-content-ru#613]
Есть ли возможность подключить ymaps модуль после иницализации блока? Дело в том, что какой конкретно модуль нужен блок решает после инициализации.

**Comment by tadatuta** [comment]
@kompolom Конечно. Позвать `modules.require()` в тот момент, когда будет понятно, что именно нужно подключать.
